### ISSUE TITLE
Shorter link previews

### ISSUE THREAD

**Issue by driesvints** [integrations/slack#1765]
**Is your feature request related to a problem? Please describe.**
Although it's nice that we get some details about an issue/pr when the link preview is shown, more often it's pretty disruptive that your entire chat window gets filled with it and it's hard to follow along with a convo.

**Describe the solution you'd like**
I wouldn't go as far as https://github.com/integrations/slack/issues/487 but what would be cool is if you could disable the issue/pr body while still maintaining all the other info. That or shorten it to about three lines or so.

**Describe alternatives you've considered**
No real alternatives are available other than totally disabling the integration.

**Additional context**
Here's a screenshot of where a link to an issue gets unfurled. It takes up the entire screen.

<img width="1383" alt="Screenshot 2023-08-28 at 08 40 45" src="https://github.com/integrations/slack/assets/594614/39567614-2215-46e0-9371-fa8c94a43cec">

**Comment by sevenseacat** [comment]
We had to disable the integration due to this - it was simply way way too noisy.

Our PRs also have a dozen or so checks on them, and they all get appended to every notification as well.

**Comment by a613** [comment]
There's a busier ticket for this over here: https://github.com/integrations/slack/issues/564

**Comment by 1DanielC** [comment]
Upvoting this!
